packages/instant-meilisearch/src/types/types.ts

```diff
@@ -2,24 +2,19 @@ import type { SearchClient } from 'instantsearch.js'
 import type { MultipleQueriesQuery as AlgoliaMultipleQueriesQuery } from '@algolia/client-search'
 
 import type {
-  SearchParams as MeiliSearchParams,
-  SearchResponse as MeiliSearchResponse,
-  MultiSearchResponse as MeilisearchMultiSearchResponse,
+  MultiSearchQuery as MeiliSearchMultiSearchParams,
+  MultiSearchResult,
 } from 'meilisearch'
 
-export type {
-  AlgoliaMultipleQueriesQuery,
-  MeilisearchMultiSearchResponse,
-  MeiliSearchParams,
-  MeiliSearchResponse,
-}
+export type { AlgoliaMultipleQueriesQuery, MultiSearchResult }
 export type { SearchResponse as AlgoliaSearchResponse } from '@algolia/client-search'
 
 export type {
   Filter,
   FacetDistribution,
   MeiliSearch,
   FacetStats as MeiliFacetStats,
+  MultiSearchQuery as MeiliSearchMultiSearchParams,
```

Comment:
Same here, it seems weird 🤔 
Upper you're importing the following:
`MultiSearchQuery as MeiliSearchMultiSearchParams,`
So can't you just import and export `MultiSearchQuery` ?

Reply:
Same as above, it's to avoid renaming variables in an unrelated PR. But I definitely agree with you